app/controllers/chargeback_controller.rb

```diff
@@ -195,6 +212,9 @@ def cb_rate_edit
           end
           @sb[:rate_details].each do |detail|
             detail.errors.each { |field, msg| add_flash("'#{detail.description}' #{field.to_s.capitalize} #{msg}", :error) }
+            detail.chargeback_tiers.each do |tier|
+              tier.errors.each { |field, msg| add_flash("'#{detail.description}' #{field.to_s.capitalize} #{msg}", :error) }
```

Comment:
Maybe start with a method that takes a single error and does what's in the block.  If that method needs to handle other types eventually, we can move that logic to a presenter.